src/client/pages/OfferNew/utils.ts

```diff
@@ -232,6 +232,12 @@ export const isSwedishApartment = (
 ): details is SwedishApartmentQuoteDetails =>
   details.__typename === 'SwedishApartmentQuoteDetails'
 
+export const isBRF = (
```

Comment:
Should this be namned:

```suggestion
export const isSwedishBRF = (
```